backend/packages/Upgrade/src/api/controllers/AnalyticsController.ts

```diff
@@ -283,13 +283,14 @@ export class AnalyticsController {
    *          '200':
    *            description: Get CSV files
    */
-  @Post('/csv')
+  @Get('/csv')
   public async downloadCSV(
-    @Body({ validate: true })
-    csvInfo: DataExportValidator,
-    @Req() request: AppRequest
+    @QueryParams()
+    param: DataExportValidator,
```

Comment:
can this be `params` instead of `param`. That's my only comment! Otherwise looks good to me.